chart/openfaas/values.yaml

```diff
@@ -201,16 +207,16 @@ ingressOperator:
     requests:
       memory: "25Mi"
 
-nodeSelector: {}
+nodeSelector: { }
 
-tolerations: []
+tolerations: [ ]
 
-affinity: {}
+affinity: { }
```

Comment:
Can you change `{ }` back to `{}` which is what I'm used to seeing in every other helm chart.

Was this a linter that you used?

Reply:
yep done...  i use  intellj